### PR TITLE
added @discardableResult to func stub for swift 3

### LINKED ISSUE #202: Needs @discardableResult added to func stub
Xcode 8 throws tons of warnings along the lines of "Result of call to 'stub(condition:response:)' is unused".

Since ignoring the return value is totally fine if you don't intend to call removeStub, you should mark the function with @discardableResult and save us from the warning spam.

### PR DESCRIPTION
**Pull request by mrkite**
fixes #202

### CODE REVIEW

**Comment by AliSoftware** [comment]
Cool! Could you please also add an entry in the `CHANGELOG` for this?

File: OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift [hunk 1]
```diff
@@ -84,7 +84,7 @@ public func fixture(filePath: String, status: Int32 = 200, headers: [NSObject: A
  *            and can be later used to remove it with `removeStub:`
  */
 #if swift(>=3.0)
-public func stub(condition: @escaping OHHTTPStubsTestBlock, response: @escaping OHHTTPStubsResponseBlock) -> OHHTTPStubsDescriptor {
+@discardableResult public func stub(condition: @escaping OHHTTPStubsTestBlock, response: @escaping OHHTTPStubsResponseBlock) -> OHHTTPStubsDescriptor {
```
Comment (AliSoftware) [inline OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift]: Just nitpicking here, but as a a code style preference, I'd have put the `@discardableResult` attribute on its own line (just before the function declaration line)

**Comment by mrkite** [comment]
Looks like Travis failed on a running time test, not due to the PR itself.

**Comment by AliSoftware** [comment]
Travis-CI builds are currently failing due to a bug in the latest version of `xcpretty`: https://github.com/supermarin/xcpretty/issues/241 so it's unrelated to `OHHTTPStubs` and the changes in that PR indeed.

**Comment by janiduw** [comment]
@AliSoftware when will this fix be available ?, Thanks

**Comment by AliSoftware** [comment]
I guess once xcpretty gets fixed (which the author promised to do by end of week)

**Comment by janiduw** [comment]
@AliSoftware Thanks for the quick response and for the great library

**Comment by janiduw** [comment]
@AliSoftware seems that Travis CI is now working

**Comment by AliSoftware** [comment]
Merged via 88cf9b2133ec086de5c51e736a2470aa4763bcfa

Thx and sorry for the delay!

**Pull request closed without merging**